### PR TITLE
Gehansen/54/copy data button

### PR DESCRIPTION
**Pull request by GraceHansen09**
I created a copy data button in the data view box, which copies the tree ring widths and years to the clipboard. In addition, I also moved the mm label to the header of "Length" and added trailing zeros so that all lengths would have the same number of characters.

### CODE REVIEW

File: leaflet-treering.js [hunk 1]
```diff
@@ -2386,6 +2390,14 @@ function ViewData(Lt) {
...
             length = 9.998;
           }
+
+          //Format length number into a string with trailing zeros
+          lengthAsAString = String(length);
+          while(lengthAsAString.length<5)
+          {
+            lengthAsAString+='0';
+          }
+
```
Comment (cmcfadden) [inline leaflet-treering.js:2400]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padEnd

Comment (cmcfadden) [reply leaflet-treering.js:2400]: using that method should make things a little more readable

File: leaflet-treering.js [hunk 2]
```diff
@@ -2459,6 +2478,18 @@ function ViewData(Lt) {
       });
     });
   },
+  /**
+   * copy text to clipboard
+   * @function enable
+   */
+  copyToClipboard = function(allData){
```
Comment (cmcfadden) [inline leaflet-treering.js:2485]: @thorn573 can chime in, but I think we want to have global functions at the bottom of the file, and stick to the convention the app is using, so 
`function copyToClipboard(allData) {} `

Comment (thorn573) [reply leaflet-treering.js:2485]: There's an example of global/helper functions with current convention under branch 'insert-break-point-fix'.

File: leaflet-treering.js (hunk 1 above)
Comment (GraceHansen09) [reply leaflet-treering.js:2400]: I just changed it thank you for the tip!

File: leaflet-treering.js (hunk 2 above)
Comment (GraceHansen09) [reply leaflet-treering.js:2485]: I changed the format and moved it to the bottom of the file! is the insert-break-point branch going to be merged soon?

Comment (thorn573) [reply leaflet-treering.js:2485]: It won't be merged soon as it's been deemed not a priority. I'm going to keep it around for whenever insert break point becomes a priority, but it'll just live in branch limbo for awhile.

**Comment by cmcfadden** [comment]
This looks good! Go ahead and merge.

**Pull request merged**